src/community/script/core/src/test/java/org/geoserver/script/app/AppTest.java

```diff
@@ -33,7 +32,7 @@ public void testSimple() throws Exception {
         FileUtils.copyURLToFile(
             getClass().getResource("main-helloWorld."+ext), new File(app, "main."+ext));
     
-        MockHttpServletResponse resp = getAsServletResponse("/script/apps/foo/main."+ext);
+        MockHttpServletResponse resp = getAsServletResponse("/rest/apps/foo/main."+ext);
```

Comment:
I think this should be ``/rest/script/apps``, in which case the app controller is also wrong. Did you test in an actual running geoserver with script installed (from before the REST update) before changing this?

Reply:
Yes. I tested with 2.9.x which is why this needed to be changed.  If not it interferes with the Script Controller at the same endpoint.

Reply:
To be clear, what was the endpoint this used in 2.9?

Reply:
Tested this with 2.9.x and needed to use this path to avoid conflicting with the Script endpoints.

Reply:
If this is a change from 2.9/2.10/2.11 then it is an unacceptable API break. What is the actual issue that required you to change this path, in a bit more detail?

Reply:
AppFinder has become AppController and this was the path in the Restlet version.
```
<!-- Scripting Apps -->
        <entry>
            <key><value>/apps/{app}</value></key>
            <value>appFinder</value>
        </entry>
        <entry>
            <key><value>/apps/{app}/</value></key>
            <value>appFinder</value>
        </entry>
```